uber/models/attendee.py

```diff
@@ -755,20 +756,34 @@ def worked_hours(self):
         return weighted_hours + self.nonshift_hours
 
     def requested(self, department):
-        return department in self.requested_depts_ints
+        return department in self.requested_depts
 
     def assigned_to(self, department):
-        return int(department or 0) in self.assigned_depts_ints
+        return department in self.assigned_depts
+
+    def has_role_in(self, department):
```

Comment:
Actually on further reflection, this method should probably exist but take the role description as a second parameter and the dept name as a first parameter.  Let me explain:

I mentioned in a comment above how we want to be able to handle a case like "email everyone assigned to Tech Ops" which is an actual email we configure at https://github.com/magfest/magprime/blob/master/magprime/automated_emails.py#L50

There's another case we want to handle, which is a case like "email everyone marked as trusted in the Staff Suite", which is also an actual email we have at https://github.com/magfest/magprime/blob/master/magprime/automated_emails.py#L75

Now that we have more flexible roles, "trusted" would probably become something like the "Manager" role for cases like this, but the same thing applies.  This ``has_role_in`` method therefore would be the most useful if it allowed us to write code like

```python
a.has_role_in('Tech Ops', 'Manager')
```

in our ``AutomatedEmail`` declarations.

Reply:
Yeah, there's a lot of places in this pull request that are unfinished or otherwise not finalized. I'll probably have to make changes to different methods once I start wading into the UI code. I mentioned this in another comment, but I've decided to keep the calculated value for circumstances like this.

I don't want to reference departments or roles by name anywhere in the code, because those will be user-modifiable. Since we've never had roles before (other than "trusted") I don't think I'll need to reference specific roles in the code at this point. However, we *DO* have references to specific departments all over the place.

In those places it will look like this:
```python
a.has_role_in(c.TECH_OPS)
```